Makefile

```diff
@@ -47,10 +47,8 @@ test-clean:
 
 .PHONY: lint
 lint:
-ifeq (, $(shell which golangci-lint))
-	./hack/install-golangcilint.sh
-endif
-	golangci-lint run
+	if [ ! -f ${GOPATH}/bin/golangci-lint ]; then ./hack/install-golangcilint.sh; fi
```

Comment:
This assumes that `golangci-lint` is in `$GOPATH/bin`. If it has been installed elsewhere (e.g. by [using the homebrew tap](https://github.com/golangci/golangci-lint#macos), this will run the script anyways. OTOH,`hack/install-golangcilint.sh` assumes that `$GOPATH/bin` is in `$PATH` 🤷‍♂ 
How about adding `$GOPATH/bin` to `$PATH` in `hack/install-golangcilint.sh`?

Reply:
Oh, that's too late :(
Because `hack/install-golangcilint.sh` doesn't _run_ `golangci-lint`. The path manipulation would have to be done in the `Makefile`. Let's not change it in the `Makefile` and let's keep your approach.

Reply:
Yeah... It's messy :(